environments/notifications/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "environments-in-app-messaging",
```

Comment:
Maybe just call this `environments-notifications` to keep this consistent with the folder name.

Alternatively, we can just change the folder name from `notifications` to `in-app-messaging`.

Reply:
IMO renaming the folder to `in-app-messaging` makes sense, since that's what we'll name our AWS account.

Reply:
+1 we should stay aligned with the AWS Amplify general categories, which in this case is in-app-messagine:
https://docs.amplify.aws/lib/in-app-messaging/overview/q/platform/js/

Reply:
As we do not have any other planned components under notifications category, I can remove notifications folder and just move the in-app-messaging folder to under environments.

Reply:
Changed to `environments-notifications` in latest commit.

Reply:
oh did not load above comments earlier, 
will change to in-app-messaging

Reply:
Changed folder name to `in-app-messaging`